api/database/seeders/DatabaseSeeder.php

```diff
@@ -184,7 +184,6 @@ private function seedPoolCandidate(User $user, Pool $pool)
         PoolCandidate::factory()->count(1)->sequence(fn () => [
             'pool_id' => $pool->id,
             'user_id' => $user->id,
-            'expected_salary' => $user->expected_salary
         ])->for($user)->afterCreating(function (PoolCandidate $candidate) use ($user) {
             // match arrays from the user
             $candidate->expectedClassifications()->sync($user->expectedClassifications()->pluck('classifications.id')->toArray());
```

Comment:
Don't forget to remove expected classifications and cmo assets.

Reply:
aye aye https://github.com/GCTC-NTGC/gc-digital-talent/pull/4266/commits/00f10ef68bf4685da7c1660f54c71b59a60c1b9c